2023-12-28T21:01:03.744Z 2023-12-28T21:01:03.744Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:03.841Z INIT_REPORT Init Duration: 766.09 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:01:03.841Z START RequestId: 3bdf06fc-9b05-5970-8653-58fb7f48f3c9 Version: $LATEST

2023-12-28T21:01:03.857Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:01:03.857Z END RequestId: 3bdf06fc-9b05-5970-8653-58fb7f48f3c9

2023-12-28T21:01:03.857Z REPORT RequestId: 3bdf06fc-9b05-5970-8653-58fb7f48f3c9	Duration: 781.72 ms	Billed Duration: 782 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	
XRAY TraceId: 1-658de20e-2603edfae5454c70c5ac4881	SegmentId: 5fe246d742cce1d4	Sampled: true	

2023-12-28T21:06:03.162Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:09:26.130Z 2023-12-28T21:09:26.130Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:09:26.243Z INIT_REPORT Init Duration: 733.53 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:09:26.243Z START RequestId: 665886ec-2ec4-5c6f-9181-d8ebc8fcf08d Version: $LATEST

2023-12-28T21:09:26.243Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:09:26.243Z END RequestId: 665886ec-2ec4-5c6f-9181-d8ebc8fcf08d

2023-12-28T21:09:26.243Z REPORT RequestId: 665886ec-2ec4-5c6f-9181-d8ebc8fcf08d	Duration: 734.29 ms	Billed Duration: 735 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:09:26.264Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:09:28.750Z 2023-12-28T21:09:28.750Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:09:28.846Z INIT_REPORT Init Duration: 742.34 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:09:28.846Z START RequestId: 25375d8e-0474-51b0-9ae7-0389e2333d52 Version: $LATEST

2023-12-28T21:09:28.847Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:09:28.847Z END RequestId: 25375d8e-0474-51b0-9ae7-0389e2333d52

2023-12-28T21:09:28.847Z REPORT RequestId: 25375d8e-0474-51b0-9ae7-0389e2333d52	Duration: 743.33 ms	Billed Duration: 744 ms	Memory Size: 512 MB	Max Memory Used: 40 MB	

2023-12-28T21:11:03.371Z 2023-12-28T21:11:03.371Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:11:03.393Z INIT_REPORT Init Duration: 224.71 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:11:09.147Z 2023-12-28T21:11:09.147Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:11:09.284Z INIT_REPORT Init Duration: 792.62 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:11:09.284Z START RequestId: f626a42f-fb4e-53ce-92b0-fa281de8a788 Version: $LATEST

2023-12-28T21:11:09.285Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:11:09.285Z END RequestId: f626a42f-fb4e-53ce-92b0-fa281de8a788

2023-12-28T21:11:09.285Z REPORT RequestId: f626a42f-fb4e-53ce-92b0-fa281de8a788	Duration: 793.48 ms	Billed Duration: 794 ms	Memory Size: 512 MB	Max Memory Used: 39 MB	

2023-12-28T21:11:13.053Z 2023-12-28T21:11:13.053Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:11:13.164Z INIT_REPORT Init Duration: 724.63 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:11:13.164Z START RequestId: d70b409d-3a07-567a-b6df-1e9adcb76a98 Version: $LATEST

2023-12-28T21:11:13.165Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:11:13.165Z END RequestId: d70b409d-3a07-567a-b6df-1e9adcb76a98

2023-12-28T21:11:13.165Z REPORT RequestId: d70b409d-3a07-567a-b6df-1e9adcb76a98	Duration: 741.59 ms	Billed Duration: 742 ms	Memory Size: 512 MB	Max Memory Used: 38 MB	

2023-12-28T21:14:30.907Z 2023-12-28T21:14:30.907Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:14:30.970Z INIT_REPORT Init Duration: 712.03 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:14:30.970Z START RequestId: e4b53cde-5e38-579f-9b59-62d9b94cc001 Version: $LATEST

2023-12-28T21:14:30.987Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:14:30.987Z END RequestId: e4b53cde-5e38-579f-9b59-62d9b94cc001

2023-12-28T21:14:30.987Z REPORT RequestId: e4b53cde-5e38-579f-9b59-62d9b94cc001	Duration: 729.35 ms	Billed Duration: 730 ms	Memory Size: 512 MB	Max Memory Used: 40 MB	

2023-12-28T21:11:03.067Z 2023-12-28T21:11:03.067Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:11:03.147Z INIT_REPORT Init Duration: 755.23 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:11:03.147Z START RequestId: 4386a0af-6c01-505e-9901-fea88a9bfa95 Version: $LATEST

2023-12-28T21:11:03.148Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:11:03.148Z END RequestId: 4386a0af-6c01-505e-9901-fea88a9bfa95

2023-12-28T21:11:03.148Z REPORT RequestId: 4386a0af-6c01-505e-9901-fea88a9bfa95	Duration: 756.47 ms	Billed Duration: 757 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	
XRAY TraceId: 1-658de466-335cbdf066567665194ba632	SegmentId: 5e1128af7dd10836	Sampled: true	

2023-12-28T21:14:28.008Z 2023-12-28T21:14:28.008Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:14:28.090Z INIT_REPORT Init Duration: 762.31 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:14:28.090Z START RequestId: 11aabe9e-175a-54e5-b9fd-e099f028c37b Version: $LATEST

2023-12-28T21:14:28.091Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:14:28.091Z END RequestId: 11aabe9e-175a-54e5-b9fd-e099f028c37b

2023-12-28T21:14:28.091Z REPORT RequestId: 11aabe9e-175a-54e5-b9fd-e099f028c37b	Duration: 763.20 ms	Billed Duration: 764 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:01:09.693Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:01:09.899Z 2023-12-28T21:01:09.899Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:09.930Z INIT_REPORT Init Duration: 236.67 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:01:12.444Z 2023-12-28T21:01:12.444Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:12.540Z INIT_REPORT Init Duration: 744.27 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:01:12.540Z START RequestId: 32b30e7a-45e0-5678-9db8-507da1b761b1 Version: $LATEST

2023-12-28T21:01:12.541Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:01:12.541Z END RequestId: 32b30e7a-45e0-5678-9db8-507da1b761b1

2023-12-28T21:01:12.541Z REPORT RequestId: 32b30e7a-45e0-5678-9db8-507da1b761b1	Duration: 745.97 ms	Billed Duration: 746 ms	Memory Size: 512 MB	Max Memory Used: 38 MB	

2023-12-28T21:06:11.243Z 2023-12-28T21:06:11.243Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:06:11.320Z INIT_REPORT Init Duration: 725.89 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:06:11.320Z START RequestId: 07f82e9b-6101-5f0d-b757-709c84830174 Version: $LATEST

2023-12-28T21:06:11.322Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:06:11.322Z END RequestId: 07f82e9b-6101-5f0d-b757-709c84830174

2023-12-28T21:06:11.322Z REPORT RequestId: 07f82e9b-6101-5f0d-b757-709c84830174	Duration: 727.30 ms	Billed Duration: 728 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:06:12.064Z 2023-12-28T21:06:12.064Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:06:12.184Z INIT_REPORT Init Duration: 790.19 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:06:12.184Z START RequestId: ee25ee6e-6bb8-5d0e-a88c-b0698e3a8265 Version: $LATEST

2023-12-28T21:06:12.185Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:06:12.185Z END RequestId: ee25ee6e-6bb8-5d0e-a88c-b0698e3a8265

2023-12-28T21:06:12.185Z REPORT RequestId: ee25ee6e-6bb8-5d0e-a88c-b0698e3a8265	Duration: 790.92 ms	Billed Duration: 791 ms	Memory Size: 512 MB	Max Memory Used: 39 MB	

2023-12-28T21:06:12.217Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:06:12.430Z 2023-12-28T21:06:12.430Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:06:12.447Z INIT_REPORT Init Duration: 229.95 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:09:27.145Z 2023-12-28T21:09:27.145Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:09:27.206Z INIT_REPORT Init Duration: 698.06 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:09:27.206Z START RequestId: fb4c8b29-4f12-551d-a573-d52ea95449ff Version: $LATEST

2023-12-28T21:09:27.245Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:09:27.245Z END RequestId: fb4c8b29-4f12-551d-a573-d52ea95449ff

2023-12-28T21:09:27.245Z REPORT RequestId: fb4c8b29-4f12-551d-a573-d52ea95449ff	Duration: 736.28 ms	Billed Duration: 737 ms	Memory Size: 512 MB	Max Memory Used: 39 MB	

2023-12-28T21:09:27.950Z 2023-12-28T21:09:27.950Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:09:28.066Z INIT_REPORT Init Duration: 738.96 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:09:28.066Z START RequestId: 54595965-402d-5340-83ba-0762a66b198b Version: $LATEST

2023-12-28T21:09:28.067Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:09:28.067Z END RequestId: 54595965-402d-5340-83ba-0762a66b198b

2023-12-28T21:09:28.067Z REPORT RequestId: 54595965-402d-5340-83ba-0762a66b198b	Duration: 739.72 ms	Billed Duration: 740 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:19:26.213Z 2023-12-28T21:19:26.213Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:19:26.333Z INIT_REPORT Init Duration: 783.60 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:19:26.333Z START RequestId: 69620930-80c3-57ef-9bca-573768543313 Version: $LATEST

2023-12-28T21:19:26.334Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:19:26.334Z END RequestId: 69620930-80c3-57ef-9bca-573768543313

2023-12-28T21:19:26.334Z REPORT RequestId: 69620930-80c3-57ef-9bca-573768543313	Duration: 784.33 ms	Billed Duration: 785 ms	Memory Size: 512 MB	Max Memory Used: 39 MB	

2023-12-28T21:01:02.752Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:01:03.032Z 2023-12-28T21:01:03.032Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:03.056Z INIT_REPORT Init Duration: 304.51 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:01:09.560Z 2023-12-28T21:01:09.560Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:09.660Z INIT_REPORT Init Duration: 767.81 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:01:09.660Z START RequestId: e5b4381b-356c-561e-b745-661cd138f1d7 Version: $LATEST

2023-12-28T21:01:09.661Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:01:09.661Z END RequestId: e5b4381b-356c-561e-b745-661cd138f1d7

2023-12-28T21:01:09.661Z REPORT RequestId: e5b4381b-356c-561e-b745-661cd138f1d7	Duration: 768.76 ms	Billed Duration: 769 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	
XRAY TraceId: 1-658de214-4dac80eff368bdf3518dad27	SegmentId: 391d5c1558e918e5	Sampled: true	

2023-12-28T21:06:03.364Z 2023-12-28T21:06:03.364Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:06:03.389Z INIT_REPORT Init Duration: 227.02 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:06:09.142Z 2023-12-28T21:06:09.142Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:06:09.220Z INIT_REPORT Init Duration: 746.25 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:06:09.220Z START RequestId: 2acfda47-fd48-54ed-96e2-844018c8eb86 Version: $LATEST

2023-12-28T21:06:09.221Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:06:09.221Z END RequestId: 2acfda47-fd48-54ed-96e2-844018c8eb86

2023-12-28T21:06:09.221Z REPORT RequestId: 2acfda47-fd48-54ed-96e2-844018c8eb86	Duration: 747.10 ms	Billed Duration: 748 ms	Memory Size: 512 MB	Max Memory Used: 39 MB	

2023-12-28T21:09:30.891Z 2023-12-28T21:09:30.891Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:09:31.045Z INIT_REPORT Init Duration: 794.40 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:09:31.045Z START RequestId: 9367867f-ab91-5018-b209-d8784dd9dad3 Version: $LATEST

2023-12-28T21:09:31.046Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:09:31.046Z END RequestId: 9367867f-ab91-5018-b209-d8784dd9dad3

2023-12-28T21:09:31.046Z REPORT RequestId: 9367867f-ab91-5018-b209-d8784dd9dad3	Duration: 795.13 ms	Billed Duration: 796 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:14:26.068Z 2023-12-28T21:14:26.068Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:14:26.146Z INIT_REPORT Init Duration: 727.37 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:14:26.146Z START RequestId: d55a03ee-d7f7-50c6-be43-5de930927758 Version: $LATEST

2023-12-28T21:14:26.147Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:14:26.147Z END RequestId: d55a03ee-d7f7-50c6-be43-5de930927758

2023-12-28T21:14:26.147Z REPORT RequestId: d55a03ee-d7f7-50c6-be43-5de930927758	Duration: 728.34 ms	Billed Duration: 729 ms	Memory Size: 512 MB	Max Memory Used: 36 MB	

2023-12-28T21:19:26.358Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:01:11.619Z 2023-12-28T21:01:11.619Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:11.740Z INIT_REPORT Init Duration: 798.71 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:01:11.740Z START RequestId: ec99c622-3950-54f2-8b55-9825dd064e4b Version: $LATEST

2023-12-28T21:01:11.740Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:01:11.740Z END RequestId: ec99c622-3950-54f2-8b55-9825dd064e4b

2023-12-28T21:01:11.740Z REPORT RequestId: ec99c622-3950-54f2-8b55-9825dd064e4b	Duration: 799.51 ms	Billed Duration: 800 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:01:12.572Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:01:12.778Z 2023-12-28T21:01:12.778Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:12.805Z INIT_REPORT Init Duration: 232.56 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:09:26.464Z 2023-12-28T21:09:26.464Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:09:26.493Z INIT_REPORT Init Duration: 229.25 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:09:29.870Z 2023-12-28T21:09:29.870Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:09:29.967Z INIT_REPORT Init Duration: 720.45 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:09:29.967Z START RequestId: 0efeefb1-2fb4-58fd-8f38-b1b6c9fc941f Version: $LATEST

2023-12-28T21:09:29.968Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:09:29.968Z END RequestId: 0efeefb1-2fb4-58fd-8f38-b1b6c9fc941f

2023-12-28T21:09:29.968Z REPORT RequestId: 0efeefb1-2fb4-58fd-8f38-b1b6c9fc941f	Duration: 721.55 ms	Billed Duration: 722 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:11:13.832Z 2023-12-28T21:11:13.832Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:11:13.944Z INIT_REPORT Init Duration: 759.86 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:11:13.944Z START RequestId: 6b29b963-43c0-5904-a7a0-e764a57165e6 Version: $LATEST

2023-12-28T21:11:13.945Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:11:13.945Z END RequestId: 6b29b963-43c0-5904-a7a0-e764a57165e6

2023-12-28T21:11:13.945Z REPORT RequestId: 6b29b963-43c0-5904-a7a0-e764a57165e6	Duration: 761.03 ms	Billed Duration: 762 ms	Memory Size: 512 MB	Max Memory Used: 39 MB	

2023-12-28T21:14:26.368Z 2023-12-28T21:14:26.368Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:14:26.400Z INIT_REPORT Init Duration: 234.53 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:06:02.989Z 2023-12-28T21:06:02.989Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:06:03.140Z INIT_REPORT Init Duration: 766.93 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:06:03.140Z START RequestId: 6f13c206-90f8-550d-977d-9c017597cb74 Version: $LATEST

2023-12-28T21:06:03.141Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:06:03.141Z END RequestId: 6f13c206-90f8-550d-977d-9c017597cb74

2023-12-28T21:06:03.141Z REPORT RequestId: 6f13c206-90f8-550d-977d-9c017597cb74	Duration: 767.91 ms	Billed Duration: 768 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	
XRAY TraceId: 1-658de33a-e05ada206df1007dd7f8473a	SegmentId: 7491e78b27c48371	Sampled: true	

2023-12-28T21:14:26.165Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:14:28.827Z 2023-12-28T21:14:28.827Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:14:28.906Z INIT_REPORT Init Duration: 781.52 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:14:28.906Z START RequestId: e7462bdb-41e6-56ab-8242-75dca69512ed Version: $LATEST

2023-12-28T21:14:28.907Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:14:28.907Z END RequestId: e7462bdb-41e6-56ab-8242-75dca69512ed

2023-12-28T21:14:28.907Z REPORT RequestId: e7462bdb-41e6-56ab-8242-75dca69512ed	Duration: 782.48 ms	Billed Duration: 783 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:14:29.894Z 2023-12-28T21:14:29.894Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:14:29.990Z INIT_REPORT Init Duration: 732.30 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:14:29.990Z START RequestId: 2a422d39-eeb6-56e9-9c53-733e09c202af Version: $LATEST

2023-12-28T21:14:29.991Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:14:29.991Z END RequestId: 2a422d39-eeb6-56e9-9c53-733e09c202af

2023-12-28T21:14:29.991Z REPORT RequestId: 2a422d39-eeb6-56e9-9c53-733e09c202af	Duration: 733.03 ms	Billed Duration: 734 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	

2023-12-28T21:01:10.743Z 2023-12-28T21:01:10.743Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:01:10.856Z INIT_REPORT Init Duration: 751.56 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:01:10.856Z START RequestId: 68c4f9c9-d16a-5e3b-9d4b-a7ce353c5482 Version: $LATEST

2023-12-28T21:01:10.857Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:01:10.857Z END RequestId: 68c4f9c9-d16a-5e3b-9d4b-a7ce353c5482

2023-12-28T21:01:10.857Z REPORT RequestId: 68c4f9c9-d16a-5e3b-9d4b-a7ce353c5482	Duration: 753.03 ms	Billed Duration: 754 ms	Memory Size: 512 MB	Max Memory Used: 41 MB	
XRAY TraceId: 1-658de216-201f99f5378f397c3f1987b1	SegmentId: 6c08afd273d4b75f	Sampled: true	

2023-12-28T21:11:03.168Z INIT_START Runtime Version: nodejs:20.v13	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:6fb7e564fae3347add879359ec92013de1e84ce02488f36ec21448277c28707c

2023-12-28T21:19:26.572Z 2023-12-28T21:19:26.572Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:19:26.600Z INIT_REPORT Init Duration: 241.95 ms	Phase: init	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:19:28.151Z 2023-12-28T21:19:28.151Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Error","errorMessage":"Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs\nDid you mean to import aws-sdk/clients/dynamodb.js?","code":"ERR_MODULE_NOT_FOUND","url":"file:///var/task/node_modules/aws-sdk/clients/dynamodb","stack":["Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/aws-sdk/clients/dynamodb' imported from /var/task/src/syncBalance/listener.mjs","Did you mean to import aws-sdk/clients/dynamodb.js?","    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)","    at new NodeError (node:internal/errors:406:5)","    at finalizeResolution (node:internal/modules/esm/resolve:238:11)","    at moduleResolve (node:internal/modules/esm/resolve:850:10)","    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:972:12)","    at defaultResolve (node:internal/modules/esm/resolve:1091:79)","    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)","    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)","    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:228:38)","    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)","    at link (node:internal/modules/esm/module_job:84:36)"]}

2023-12-28T21:19:28.234Z INIT_REPORT Init Duration: 734.72 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError

2023-12-28T21:19:28.234Z START RequestId: e311fe27-b4fe-5eca-b4c3-a0205c601bdc Version: $LATEST

2023-12-28T21:19:28.250Z Unknown application error occurred
Runtime.Unknown

2023-12-28T21:19:28.250Z END RequestId: e311fe27-b4fe-5eca-b4c3-a0205c601bdc

2023-12-28T21:19:28.250Z REPORT RequestId: e311fe27-b4fe-5eca-b4c3-a0205c601bdc	Duration: 751.11 ms	Billed Duration: 752 ms	Memory Size: 512 MB	Max Memory Used: 38 MB	

